platform/darwin/src/NSExpression+MGLAdditions.mm

```diff
@@ -1153,6 +1161,15 @@ - (id)mgl_jsonExpressionObject {
                 } else if ([type isEqualToString:@"NSNumber"]) {
                     return @[@"to-number", object];
                 }
+#if TARGET_OS_IPHONE
+                else if ([type isEqualToString:@"UIColor"]) {
+                    return @[@"to-rgba", object];
```

Comment:
I had things backwards in #11011. `to-string` turns its argument into a string, but oddly enough, `to-rgba` turns a color into an array, not the other way around. So `CAST(…, 'UIColor')` would more or less correspond to the `rgba` operator, and the `to-rgba` operator would correspond to `CAST(…, 'NSArray')`. Both coercions would be useful.